ECHO-COVID19.ipynb

```diff
@@ -744,12 +601,12 @@
    },
    "outputs": [],
    "source": [
-    "mnth1 = \"0\" + str(int(mnth) + 1)\n",
-    "now = exceeds[(exceeds['MONITORING_PERIOD_END_DATE'] >= '2020-'+mnth+'-01') & (exceeds['MONITORING_PERIOD_END_DATE'] < '2020-'+mnth1+'-01')] \n",
+    "latest = exceeds[(exceeds['MONITORING_PERIOD_END_DATE'] >= '2020')]\n",
     "\n",
-    "pollutants = now.groupby(['PARAMETER_DESC'])[['MONITORING_PERIOD_END_DATE']].count()\n",
+    "pollutants = latest.groupby(['PARAMETER_DESC'])[['MONITORING_PERIOD_END_DATE']].count()\n",
     "pollutants = pollutants.rename(columns={'MONITORING_PERIOD_END_DATE': \"Violations\"})\n",
     "pollutants = pollutants.sort_values(by='Violations', ascending=False)\n",
+    "\n",
     "fig = plt.figure(1, figsize=(20,20))\n",
     "ax = fig.add_subplot(111)\n",
     "wedges, labels, autopct = ax.pie(pollutants[\"Violations\"], labels = pollutants.index, autopct='%1.1f%%', radius = 10);#, textprops={'fontsize': 72});\n",
```

Comment:
text inside the pie chart is too small to read

Reply:
yay, figured this out!